README.md

```diff
@@ -1,98 +1,206 @@
+implementation("com.babylon.orbit2:orbit-core:<latest-version>") // <-- This module is mandatory
+implementation("com.babylon.orbit2:orbit-coroutines:<latest-version>")
+implementation("com.babylon.orbit2:orbit-rxjava2:<latest-version>")
+implementation("com.babylon.orbit2:orbit-livedata:<latest-version>")
+implementation("com.babylon.orbit2:orbit-savedstate:<latest-version>")
+
+testImplementation("com.babylon.orbit2:orbit-test:<latest-version>")
 ```
 
-A real-world redux system might look as follows:
+[![Download](https://api.bintray.com/packages/babylonpartners/maven/orbit-core/images/download.svg)](https://bintray.com/babylonpartners/maven/orbit-core/_latestVersion)
 
-``` kotlin
-data class State(val total: Int = 0)
+For detailed documentation, see:
 
-data class AddAction(val number: Int)
+- [Core module and architecture overview](orbit-2-core/README.MD)
+- [Coroutines](orbit-2-coroutines/README.md)
+- [RxJava 2](orbit-2-rxjava2/README.md)
+- [LiveData](orbit-2-livedata/README.md)
+- [Saved state](orbit-2-savedstate/README.md)
+- [Test](orbit-2-test/README.md)
 
-sealed class SideEffect {
-    data class Toast(val text: String) : SideEffect()
+### A word on DSL plugins
+
+Modules like `orbit-coroutines` or `orbit-rxjava2` contain plugins that extend
+Orbit's DSL. In order for Orbit to recognise these plugins, they need to be
+installed at runtime, like so:
+
+``` kotlin
+class MyApplication : Application() {
+    override fun onCreate() {
+        OrbitDslPlugins.register(
+            CoroutineDslPlugin,
+            RxJava2DslPlugin
+        )
+    }
 }
+```
 
-class CalculatorViewModel : OrbitViewModel<State, SideEffect> (State(), {
+The core syntax DSL plugin (`transform/sideEffect/reduce`) works out of the box
+with no explicit configuration necessary.
 
-    perform("addition")
-        .on<AddAction>()
-        .sideEffect { post(SideEffect.Toast("Adding ${event.number}")) }
-        .reduce {
-            currentState.copy(currentState.total + event.number)
-        }
+## Creating a simple Orbit 2 ViewModel
 
-    ...
-})
-```
+Using the core Orbit DSL, we can create a simple, functional ViewModel.
+
+### Define the contract
 
-And then in your activity / fragment
+First, we need to define its state and declared side effects.
 
 ``` kotlin
-// Example of injection using koin, your DI system might differ
-private val viewModel by viewModel<CalculatorViewModel>()
+data class CalculatorState(
+    val total: Int = 0
+)
 
-override fun onCreate() {
-    ...
-    addButton.setOnClickListener { viewModel.sendAction(AddAction) }
+sealed class CalculatorSideEffect {
+    data class Toast(val text: String) : CalculatorSideEffect()
 }
+```
 
-override fun onStart() {
-    viewModel.connect(this, ::handleState, ::handleSideEffect)
-}
+The only requirement here is that the objects are comparable. We also recommend
+they be immutable. Therefore we recommend using a mix of data classes, sealed
```

Comment:
better to use the word `suggest` instead of `recommend` here just to avoid it as its used in the preceding sentence